main.go

```diff
@@ -100,6 +100,15 @@ func main() {
 	}
 	defer utils.Close(lf)
 
+	endpoint := os.Getenv("SPANNER_API_ENDPOINT")
+	emulatorHost := os.Getenv("SPANNER_EMULATOR_HOST")
+	if !((endpoint == "" && (emulatorHost == "" || emulatorHost == constants.SPANNER_EMULATOR_HOST)) || (endpoint == constants.STAGING_SPANNER_API_ENDPOINT && emulatorHost == "")) {
+		fmt.Printf("\nWarning: Endpoint specified may be incorrect: \n" +
+			"For connecting to prod set SPANNER_API_ENDPOINT as blank \n" +
```

Comment:
Would it be clear to the user that we are talking about the env variable SPANNER_API_ENDPOINT?
Also suggest fixing the punctuation a bit.
```
fmt.Printf("\nWarning: Endpoint specified may be incorrect\n" +
			"For connecting to prod, run: export SPANNER_API_ENDPOINT=\"\" \n" +
			"For connecting to emulator, run: export SPANNER_EMULATOR_HOST=" + constants.SPANNER_EMULATOR_HOST + "\n" +
			"For connecting to staging, run: export SPANNER_API_ENDPOINT=" + constants.STAGING_SPANNER_API_ENDPOINT + "\n\n")
```